ClrHeapAllocationsAnalyzer.Vsix/source.extension.vsixmanifest

```diff
@@ -9,9 +9,24 @@
         <Tags>roslyn clr allocations boxing closure display performance</Tags>
     </Metadata>
     <Installation>
-        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[15.0,)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.0,)" />
+		<InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,17.0)">
+		  <ProductArchitecture>x86</ProductArchitecture>
+		</InstallationTarget>
+		<InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0,18.0)">
+		  <ProductArchitecture>amd64</ProductArchitecture>
+		</InstallationTarget>
+        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[15.0,)">
+			<ProductArchitecture>x86</ProductArchitecture>
+		</InstallationTarget>
+		<InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[17.0,18.0)">
+		  <ProductArchitecture>amd64</ProductArchitecture>
+		</InstallationTarget>
+        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.0,)">
+			<ProductArchitecture>x86</ProductArchitecture>		
```

Comment:
Can you fix up the tabs here? And use spaces instead in this entire file?

Reply:
No problem, just later.

Reply:
@mjsabby done